xml/operations-configuring-identity-identity_limitations.xml

```diff
@@ -322,8 +322,17 @@ for production use until a future OpenStack release.</li> -->
    </listitem>
    <listitem>
     <para>
-     Only WebSSO federation using the SAML method is supported in
-     &product; . OpenID-based federation is not currently supported.
+     The &productname; WebSSO function with OpenID method was tested with
+     Google OAuth 2.0 APIs, which conform to the OpenID Connect specification.
+     The interaction between Keystone and the external Identity Provider (IdP) is
```

Comment:
`Keystone` should be `keystone` lower case

Reply:
Done